webroot/src/models/Licensee/Licensee.model.spec.ts

```diff
@@ -115,16 +116,16 @@ describe('Licensee model', () => {
 
         // Test methods
         expect(licensee.nameDisplay()).to.equal(`${data.firstName} ${data.lastName}`);
-        expect(licensee.residenceLocation()).to.equal('');
+        expect(licensee.residenceLocation()).to.equal('Unknown');
         expect(licensee.dobDisplay()).to.equal('1/1/2020');
         expect(licensee.ssnMaskedFull()).to.equal(data.ssn);
         expect(licensee.ssnMaskedPartial()).to.equal('test-ss-ssn');
         expect(licensee.lastUpdatedDisplay()).to.equal('1/1/2020');
         expect(licensee.lastUpdatedDisplayRelative()).to.be.a('string').that.is.not.empty;
         expect(licensee.getStateListDisplay([])).to.equal('');
-        expect(licensee.licenseStatesDisplay()).to.equal('');
-        expect(licensee.privilegeStatesAllDisplay()).to.equal('');
-        expect(licensee.privilegeStatesDisplay()).to.equal('');
+        expect(licensee.licenseStatesDisplay()).to.equal('Unknown, Unknown +');
```

Comment:
What's "Unknown +"?

Reply:
The " +" is the short-display version of the states. If there are more than 2 states to display, the designs call for "State1, State2 +".